reports/2022.html

```diff
@@ -141,9 +141,9 @@ <h3>Table of Contents</h3>
             </tr>
             <tr>
               <th><a href="#themeing">Themeing</a></th>
+              <td><a href="https://github.com/WICG/webcomponents/issues/864">WICG/webcomponents#864</a></td>
               <td></td>
```

Comment:
What's the priority level for this?

Reply:
Yeah, I was thinking that maybe we [drop that column](https://github.com/w3c/webcomponents-cg/issues/43#issuecomment-1221625315) for this year's report?  In favor of our new outline per #53